### DISCUSSION TITLE
longerking-blogs/2023/07/16/my-site-starter/

### DISCUSSION THREAD

**Discussion by giscus** [longerking/longerking-blogs#10] [General]
# longerking-blogs/2023/07/16/my-site-starter/

Just Work for Life,  Be Your Longer King!

https://longerking.github.io/longerking-blogs/2023/07/16/my-site-starter/

**Comment by longerking** [comment]
简洁干净的博客主题 hexo keep theme，要是能同步部署到gitee page就好了
